pygeo/mphys/mphys_dvgeo.py

```diff
@@ -215,6 +219,9 @@ def nom_addPointSet(self, points, ptName, add_output=True, DVGeoName=None, **kwa
             # add an output to the om component
             self.add_output(ptName, distributed=True, val=points.flatten())
 
+        if isinstance(DVGeo, DVGeometryESP):
+            return dMax_global
```

Comment:
I prefer to not use conditional returns, can we tweak this to set `dMaxGlobal` equal to `None` if the `DVGeo` type is not `DVGeometryESP`? Then just return the variable without the if-statement.